css/style.css

```diff
@@ -0,0 +1,845 @@
+  width: 20px;
+  height: 20px;
+  align-self: center;
+  margin-right: 18px;
+}
+
+.card-skills-title .icons-skills-circle {
+  width: 20px;
+  height: 20px;
+  align-self: center;
+}
+
+.language-skill {
+  margin-right: 176px;
+  width: 225px;
+  height: 281px;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.lang-title {
+  margin-left: 12px;
+  font-family: Inter, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  align-self: center;
+  text-align: center;
+  color: #fff;
+}
+
+.tags-skills {
+  font-family: Inter, sans-serif;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 42px;
+  text-align: left;
+  color: #fff;
+  list-style: none;
+}
+
+.contact-container {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
+
+.contact-form {
+  margin-left: 24px;
+  margin-top: 130px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-family: Inter, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+}
+
+input {
+  width: 295px;
+  height: 18px;
+  border: 0;
+  margin-left: 16px;
+}
+
+input[type="email"] {
+  margin-top: 39px;
+}
+
+input[type="submit"] {
+  width: 122px;
+  height: 48px;
+  margin-top: 24px;
+  background-color: #f55800;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+}
+
+textarea {
+  margin-top: 24px;
+  padding: 12px;
+  border: 0;
+  font-family: Inter, sans-serif;
+  background: #fbf8f7;
+}
+
+.separator-line {
+  margin-top: 14px;
+  width: 684px;
+  height: 1px;
+  background: #dbd8d7;
+  border: 0;
+}
+
+.bottom-social-media {
+  width: 100%;
+  height: 72px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+}
+
+.bottom-social-media hr {
+  margin-top: 0;
+  padding-top: 0;
+  margin-bottom: 15px;
+  justify-self: flex-start;
+}
+
+.media-links {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.button_form:hover {
+  cursor: pointer;
+  color: #fff;
+  background-color: #000;
+  border: 2px solid #f55800;
+}
+
+.button:hover {
+  box-shadow:
+    0 12px 16px 0 rgba(0, 0, 0, 0.24),
+    0 17px 50px 0 rgba(0, 0, 0, 0.19);
+}
+
+@keyframes text {
+  from {
+    background-color: #cfcfcf;
+  }
+
+  to {
+    background-color: #f55800;
+  }
+}
+
+@media (max-width: 768px) {
+  .portfolio-menu {
+    display: none;
+    font-family: Inter, sans-serif;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    text-align: center;
+    align-items: center;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .head-container {
+    display: flex;
+    flex-direction: row;
+    background-color: black;
+    background-image: none;
+  }
+
+  .head-information h1 {
+    font-size: 4rem;
+  }
+
+  .header-about p {
+    margin-left: 142px;
```

Comment:
I suggest you fix the styles of your page. It uses the template, but your CSS is broken. This is how your page appears both on mobile and Desktop
<img width="1680" alt="Screenshot 2023-01-21 at 07 56 49" src="https://user-images.githubusercontent.com/48546992/213844317-ce01abbf-8197-4eef-ab7f-6bd72da1da3e.png">
<img width="425" alt="Screenshot 2023-01-21 at 07 57 05" src="https://user-images.githubusercontent.com/48546992/213844320-73cf055f-f2f8-49fc-b387-ccfca71c37b6.png">